gum/backend-darwin/gumtls-darwin.c

```diff
@@ -8,6 +8,12 @@
 
 #include <pthread.h>
 
+void
+_gum_tls_init (void)
+{
+  (void);
```

Comment:
No-op? This kind of construct should only be used for suppressing warnings about unused arguments.